pkg/lagoon/lagoon.go

```diff
@@ -20,12 +25,31 @@ type Fact struct {
 	Category    string `json:"category"`
 }
 
+type ProblemSeverityRating string
+
+type Problem struct {
+	EnvironmentId     int                   `json:"environment"`
+	Identifier        string                `json:"identifier"`
+	Version           string                `json:"version,omitempty"`
+	FixedVersion      string                `json:"fixedVersion,omitempty"`
+	Source            string                `json:"source,omitempty"`
+	Service           string                `json:"service,omitempty"`
+	Data              string                `json:"data"`
+	Severity          ProblemSeverityRating `json:"severity,omitempty"`
+	SeverityScore     float64               `json:"severityScore,omitempty"`
+	AssociatedPackage string                `json:"associatedPackage,omitempty"`
+	Description       string                `json:"description,omitempty"`
+	Links             string                `json:"links,omitempty"`
+}
+
 const SourceName string = "Shipshape"
 const FactMaxValueLength int = 300
 
 var ApiBaseUrl string
 var ApiToken string
```

Comment:
Are we still using these given we're now using the in-cluster token?

Reply:
Yep, just until the release of Lagoon 2.18, really - for clearing problems when none are found.

Reply:
Oh yes, you mentioned that